src/main.py

```diff
@@ -2,7 +2,7 @@
 Hey there.
 
 If you found this, send a pull request to this repo removing
-this file and I'll send you $5.
+this file and I'll send you $5,000.
```

Comment:
It's too bad GFM can't render diffs with inter-block diffs, the way commit diffs are displayed.

![image](https://cloud.githubusercontent.com/assets/1924134/7107630/db6328ac-e11f-11e4-8faf-87a20e8d62a4.png)